spec/helpers/list_space.rb

```diff
@@ -0,0 +1,9 @@
+def list_space
```

Comment:
Use of the helper super helps with readability
Could you replace lines 3 to 7 by using Space.create?